pkgs/llvm_cheriot.nix

```diff
@@ -0,0 +1,28 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+{stdenv, fetchFromGitHub, lib, cmake, ninja, lld, python3}:
+stdenv.mkDerivation rec {
+  name = "llvm-cheriot";
+  src = fetchFromGitHub {
+    owner = "CHERIoT-Platform";
+    repo = "llvm-project";
+    rev = "432ef65c96c9f8be8b7802a8bfb9bcf3609b2b79";
+    sha256 = "sha256-ddLnG1olyRYxvKJ0zKEbThq/ZnxP68X32+1ws76WeV4=";
```

Comment:
This should hash instead of sha256.

Reply:
Sorry, I don't follow. What form should this hash be in?